projects/step-frontend/src/lib/modules/timeseries/performance-view/time-selection/execution-page-time-selection.component.ts

```diff
@@ -100,6 +104,7 @@ export class ExecutionPageTimeSelectionComponent implements OnInit {
   }
 
   prepareSelectForRanger(selection: ExecutionTimeSelection): TSTimeRange | undefined {
+    console.log(selection);
```

Comment:
Console.log should be removed from the code before releasing

Reply:
In general we should remove console.log always before merging into master, for debug we usually can use break points. If this is not possible at all set a descriptive lable like:
` console.log('prepared ranger selection is:', selection);`